Applications/MmrMobile/Contracts/AlertEvent.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Runtime.Serialization;
+
+namespace HomagConnect.MmrMobile.Contracts
+{
+    public class AlertEvent : IExtensibleDataObject
+    {
+        /// <summary>
+        /// Timestamp, when the Event started
+        /// </summary>
+        [JsonProperty(Order = 1)]
+        public DateTimeOffset EventStartTime { get; set; }
```

Comment:
Do we need the prefix "Event"?

Reply:
In Helios, we explicitely use the term Event or eventdata
And as we have the topic "woodscout" (which I would not like to name, as it was to be renamend to intelliAlert once)
and we have the topic "maintenance" events. It would be fitting (in my opinion) to name it like that.